examples/fhir/MN NBS FHIR Order Message.json

```diff
@@ -0,0 +1,773 @@
+        "id": "34685122-eb17-44c0-aa2c-6a67461dcb73",
+        "status": "active",
+        "intent": "order",
+        "subject": {
+          "reference": "Patient/09a7a7cc-7456-4f09-bdf6-ebc730902fd2",
+          "type": "Patient"
+        },
+        "requester": {
+          "reference": "PractitionerRole/e3b3ff93-1c84-4b82-9a23-e26c95985bd9",
+          "type": "PractitionerRole"
+        },
+        "specimen": [
+          {
+            "reference": "Specimen/1438062e-b4d2-4f63-a13c-2dd6e5f8e3ed",
+            "type": "Specimen"
+          }
+        ],
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "54089-8",
+              "display": "Newborn screening panel American Health Information Community (AHIC)"
+            }
+          ],
+          "text": "Newborn screening panel American Health Information Community (AHIC)"
+        },
+        "authoredOn": "2023-05-06T05:00:00Z"
+      }
+    },
+    {
+      "fullUrl": "ServiceRequest/742d6734-e14c-45b3-af73-f1ee919b3e19",
+      "resource": {
+        "resourceType": "ServiceRequest",
+        "id": "742d6734-e14c-45b3-af73-f1ee919b3e19",
+        "status": "active",
+        "intent": "order",
+        "subject": {
+          "reference": "Patient/09a7a7cc-7456-4f09-bdf6-ebc730902fd2",
+          "type": "Patient"
+        },
+        "requester": {
+          "reference": "PractitionerRole/e3b3ff93-1c84-4b82-9a23-e26c95985bd9",
+          "type": "PractitionerRole"
+        },
+        "specimen": [
+          {
+            "reference": "Specimen/1438062e-b4d2-4f63-a13c-2dd6e5f8e3ed",
+            "type": "Specimen"
+          }
+        ],
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "57717-1",
+              "display": "Newborn screen card data panel"
+            }
+          ],
+          "text": "Newborn screen card data panel"
+        },
+        "authoredOn": "2023-05-06T05:00:00Z"
+      }
+    },
+    {
+      "fullUrl": "Specimen/1438062e-b4d2-4f63-a13c-2dd6e5f8e3ed",
+      "resource": {
+        "resourceType": "Specimen",
+        "id": "1438062e-b4d2-4f63-a13c-2dd6e5f8e3ed",
+        "type": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "440500007",
+              "display": "Blood spot specimen"
+            }
+          ],
+          "text": "Blood spot specimen"
+        },
+        "subject": {
+          "reference": "Patient/09a7a7cc-7456-4f09-bdf6-ebc730902fd2",
+          "type": "Patient"
+        },
+        "receivedTime": "2023-05-06T06:00:00-05:00"
+      }
+    },
+    {
+      "fullUrl": "Observation/faa6f28d-840a-41f9-9a50-a23cf89ef84b",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "faa6f28d-840a-41f9-9a50-a23cf89ef84b",
+        "status": "final",
+        "category": [
```

Comment:
I'll remove all these categories; it would be difficult to derive them from the ORM and doesn't provide much value

Reply:
Done!